wfl/Models/FoodOrder.cs

```diff
@@ -78,4 +78,9 @@ public bool CanReactivate()
             return false;
         }
     }
+
+    public class StatusOptionsRepresentation
+    {
+        public IList<string> values;
```

Comment:
Make this title case

* [ ] make parameter title case with a `{get;set;}` to make it a property and not a field